cajun-jsonapi.spec

```diff
@@ -1,14 +1,15 @@
-# cajun only ships headers, so no debuginfo package is needed
-BuildArch:      noarch
 
-Summary: A cross-platform C++ header library for JSON
-Name: cajun-jsonapi
-Version: 2.0.3
-Release: 1%{?dist}
-URL: https://github.com/cajun-jsonapi/cajun-jsonapi
-Source0: https://github.com/cajun-jsonapi/cajun-jsonapi/archive/%{version}.tar.gz
-License: BSD
-Group: System Environment/Libraries
+Name:           cajun-jsonapi
+Version:        2.0.3
+Release:        0
+Summary:        A cross-platform C++ header library for JSON
+License:        BSD-3-Clause
+Group:          Development/Libraries/C and C++
+Url:            https://github.com/cajun-jsonapi/cajun-jsonapi
+Source0:        %{name}-%{version}.tar.gz
```

Comment:
the format %{version}.tar.gz is unfortunate since the tarball is just named %{version} in that case. Can github change to %{name}-%{version}?